engine.py

```diff
@@ -301,8 +301,14 @@ def _emit_log_message(self, handler, record):
             record.message
         )
 
-        sys.stdout.write(msg_str)
-        sys.stdout.flush()
+        try:
```

Comment:
I'm starting to think it's also not ideal that we're using `stdout`/`stderr` to send python log messages back to the adobe side. I think we may be better to send them via rpc. We can discuss.